### PR TITLE
data: ankitk26

### PR DESCRIPTION
**Pull request by ankitk26**

### CODE REVIEW

File: data/ankitk26.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/ankitkr26/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/ankitk26",
+      "icon": "FaGithub"
+    },
```
Comment (priyankarpal) [inline data/ankitk26.json:20]: Hey, @ankitk26 kindly delete this comma. 
``` suggestion
    }
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/ankitk26.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+	"name": "Ankit Kumar",
+	"type": "personal",
+	"bio": "Software developer getting involved in Open Source. Music and Anime fan",
+	"avatar": "https://github.com/ankitk26.png",
```
Comment (priyankarpal) [inline data/ankitk26.json:5]: After the update, you don't need to add an `avatar` code. Just delete the line.
``` suggestion

```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**